docs/tutorials/start-a-private-network/alicebob.md

```diff
@@ -7,29 +7,33 @@ Before we generate our own keys, and start a truly unique Substrate network, let
 ## Alice Starts First
 
 Alice (or whomever is playing her) should run this command from node-template repository root.
+
 ```bash
 # Start Alice's node
 ./target/release/node-template \
   --base-path /tmp/alice \
-  --chain=local \
+  --chain local \
   --alice \
   --port 30333 \
   --ws-port 9944 \
   --rpc-port 9933 \
-  --telemetry-url ws://telemetry.polkadot.io:1024 \
-  --validator
+  --telemetry-url ws://telemetry.polkadot.io:1024
```

Comment:
```suggestion
  --telemetry-url ws://telemetry.polkadot.io:1024 \
  --validator
```
Removing `--validator` is correct because running it as `bob` and `alice` implies that they are validators.
It could be redundant to have this option, but it make sense to keep it. It will make it easier to understand that they are validators.

Reply:
> It could be redundant to have this option, but it makes sense to keep it. It will make it easier to understand that they are validating the chain.

I agree with @riusricardo Although the flag is redundant, they will need it in later steps when they use their own keys. I vote to keep the redundant flag.